bundler/spec/fixtures/projects/bundler1/git_source_with_multiple_deps/Gemfile.lock

```diff
@@ -1,5 +1,5 @@
 GIT
-  remote: git://github.com/dependabot-fixtures/elasticsearch-rails.git
+  remote: https://github.com/dependabot-fixtures/elasticsearch-rails.git
```

Comment:
Not sure that we should just change the lockfile, users might specify theirs this way, and if there is a way to make this work that would be better 🤔